packages/tools/pie-icons-webc/test/visual/pie-icons-webc.spec.ts

```diff
@@ -1,50 +1,47 @@
 import { percyWidths } from '@justeattakeaway/pie-webc-testing/src/percy/breakpoints.ts';
-import { IconChatConversation } from '@justeattakeaway/pie-icons-webc/dist/IconChatConversation';
-import { IconChatConversationLarge } from '@justeattakeaway/pie-icons-webc/dist/IconChatConversationLarge';
-
-// This ensures the component is registered in the DOM for each test
-// This is not required if your tests mount the web component directly in the tests
-test.beforeEach(async ({ mount }) => {
-    const iconChatConversation = await mount(IconChatConversation);
-    await iconChatConversation.unmount();
-    const iconChatConversationLarge = await mount(IconChatConversationLarge);
-    await iconChatConversationLarge.unmount();
-});
+import { BasePage } from '@justeattakeaway/pie-webc-testing/src/helpers/page-object/base-page.ts';
+import { icons } from '../helpers/page-object/selectors.ts';
 
 test.describe('PIE Icons Webc - Visual tests`', () => {
-    test('Regular and Large icons render', async ({ page, mount }) => {
-        await mount(IconChatConversation);
-        await mount(IconChatConversationLarge);
-
-        await percySnapshot(page, 'PIE Icons Webc - Regular and Large icons render', percyWidths);
+    test('Regular icon renders', async ({ page }) => {
+        // Arrange
+        const regularIconPage = new BasePage(page, 'icons--alcohol-filled-regular-icon-variations');
+        await regularIconPage.load();
+        await expect.soft(page.locator(icons.selectors.alcoholFilled.dataTestId).first()).toBeVisible();
+
+        // Assert
+        await percySnapshot(page, 'PIE Icons Webc - Regular icon renders', percyWidths);
     });
 
-    test('Regular and Large icons resize based on size prop', async ({ page, mount }) => {
-        await mount(IconChatConversation, {
-            props: {
-                size: 'xxl',
-            },
-        });
+    test('Large icon renders', async ({ page }) => {
+        // Arrange
+        const largeIconPage = new BasePage(page, 'icons--alcohol-filled-large-icon-variations');
+        await largeIconPage.load();
+
+        // Assert
+        await expect.soft(page.locator(icons.selectors.alcoholFilledLarge.dataTestId).first()).toBeVisible();
+        await percySnapshot(page, 'PIE Icons Webc - Large icon renders', percyWidths);
+    });
 
-        await mount(IconChatConversationLarge, {
-            props: {
-                size: 80,
-            },
-        });
+    test('Regular icon can be sized with the override CSS variable', async ({ page }) => {
+        // Arrange
+        const regularIconOverridePage = new BasePage(page, 'icons--alcohol-filled-regular-icon-with-override');
+        await regularIconOverridePage.load({ size: 'm' });
 
-        await percySnapshot(page, 'Regular and Large icons resize based on size prop', percyWidths);
+        // Assert
+        await expect.soft(page.locator(icons.selectors.alcoholFilled.dataTestId)).toBeVisible();
```

Comment:
I'm not sure why we need the `toBeVisible` assertion when we're already doing a visual test? I thought `expect.soft` means it won't stop the test if it fails, so it isn't saving us a screenshot, right?

Reply:
Same comment for all instances of this

Reply:
This is just a more robust way to ensure the page is loaded and the components are actually rendered prior to taking the screenshot 👍🏼